app/pages/preview/helpers/getServiceRecipients.js

```diff
@@ -0,0 +1,9 @@
+export const getServiceRecipients = (serviceRecipients = []) => {
```

Comment:
just on the name of this function. Any reason you went for `getServiceRecipients`? we normally use the `get` prefix when it's going to call the api to get some data. Do you think we could rename this function to something else?

Reply:
I wasn't aware that you reserved the `get` prefix for API calls and I just thought that `getServiceRecipients` was more concise than `createServiceRecipientsDict`. However, I'll rename to the latter.